### ISSUE TITLE
Automatic hide window genoil

### ISSUE THREAD

**Issue by Smokeyw** [Genoil/ZECMiner#84]
is there a way to auto start genoil-zec-miner when windows start 
and after that hide the genoil-zec-miner window so nobody know or see a thing ??

**Comment by tuzak500** [comment]
lol why? you want to run it at work place computers or smt? ... "honest".

**Comment by ginyor** [comment]
yes. :) 1. msconfig; 2. https://stackoverflow.com/questions/3677773/how-can-i-run-a-windows-batch-file-but-hide-the-command-window

**Comment by tuzak500** [comment]
hmm for been honest.. i would do it like this:

Set WshShell = CreateObject("WScript.Shell")
WshShell.Run chr(34) & "C:\Batch Files\syncfiles.bat" & Chr(34), 0
Set WshShell = Nothing

Copy the lines above to Notepad and save the file with .VBS extension. Edit the .BAT file name and path accordingly, and save the file. Double-click the .VBS file to run it.

also so you get more confident if anything fails run this bat file
:restart
start "" cmd /C startz.bat
TIMEOUT /T 3600
taskkill /f /im genoil.exe
TIMEOUT /T 2
goto restart

so if anything goes wrong it will restart the bat every hour or so..

**Comment by Smokeyw** [comment]
thanks guyz going to test it on the work place on every pc hahaha
